src/Command/Yaml/DiffCommand.php

```diff
@@ -177,35 +170,52 @@ protected function execute(InputInterface $input, OutputInterface $output)
      */
     protected function interact(InputInterface $input, OutputInterface $output)
     {
-        $validator_filename = function ($value) {
+        $io = new DrupalStyle($input, $output);
+
+        $validator_filename = function ($value) use ($io) {
             if (!strlen(trim($value))) {
-                throw new \Exception(' You must provide a valid file path.');
```

Comment:
And with the "while" if the user enters an invalid path then the system will ask again for a valid path.